trend_micro_vision_one_endpoint_security/assets/logs/trend-micro-vision-one-endpoint-security.yaml

```diff
@@ -0,0 +1,2633 @@
+id: trend-micro-vision-one-endpoint-security
+metric_id: trend-micro-vision-one-endpoint-security
+backend_only: false
+facets:
+  - groups:
+      - Geoip
+    name: City Name
+    path: network.client.geoip.city.name
+    source: log
+  - groups:
+      - Geoip
+    name: Continent Code
+    path: network.client.geoip.continent.code
+    source: log
+  - groups:
+      - Geoip
+    name: Continent Name
+    path: network.client.geoip.continent.name
+    source: log
+  - groups:
+      - Geoip
+    name: Country ISO Code
+    path: network.client.geoip.country.iso_code
+    source: log
+  - groups:
+      - Geoip
+    name: Country Name
+    path: network.client.geoip.country.name
+    source: log
+  - groups:
+      - Geoip
+    name: Subdivision ISO Code
+    path: network.client.geoip.subdivision.iso_code
+    source: log
+  - groups:
+      - Geoip
+    name: Subdivision Name
+    path: network.client.geoip.subdivision.name
+    source: log
+  - groups:
+      - Web Access
+    name: Client IP
+    path: network.client.ip
+    source: log
+  - groups:
+      - Web Access
+    name: Client Port
+    path: network.client.port
+    source: log
+  - groups:
+      - Geoip
+    name: Destination City Name
+    path: network.destination.geoip.city.name
```

Comment:
Does not seem to be a standard attribute (https://docs.datadoghq.com/standard-attributes/?product=log+management), can we namespace and remap that ?

Reply:
We tried removing the GeoIP attributes that you mentioned, but this resulted in the CI pipeline failing, as shown [here](https://github.com/DataDog/integrations-core/runs/29095080623).

Reply:
Hey @shubhamvekariya-crest, I am new to the log-integration reviewing process, and thought you had to namespace those non standard path with something like trend_micro_vision_one_endpoint_security.network.destination.geoip.city.name. But actually network.destination.geoip.city.name is already namespaced with the group value (geoip) so it should be enough, let me resolve those.